tests/src/helpers.test.js

```diff
@@ -3,19 +3,17 @@
  * @author  TheJaredWilcurt
  */
 
-jest.mock('os');
```

Comment:
I think it was related to this, where we mock out the OS module to fake which OS we are on for certain tests.

Reply:
I think I mocked out one way of checking the OS/platform, and exclusively used that way of checking for it in the library. That way I could pretend to be a different OS in the tests. But also I still needed a way to check the real OS that was running the tests, because it would have different behavior and the tests still needed to pass on all real OS's. So I used a different way to check the real OS that wasn't mocked. I think that's what was going on with `process.platform` vs `require('os').platform()`.